### PR TITLE
Slightly better error reporting when version is missing

### PR DESCRIPTION
**Pull request by sigurdm**
Inspired by https://github.com/dart-lang/pub-dev/issues/6508

### CODE REVIEW

File: lib/src/utils.dart [hunk 1]
```diff
@@ -69,10 +69,10 @@ Future<PanaProcessResult> runProc(
     process.stdout.forEach((outLine) {
       stdoutLines.add(outLine);
       remainingBytes -= outLine.length;
-      if (remainingBytes < 0) {
-        wasOutputExceeded = true;
-        killProc('Output exceeded $maxOutputBytes bytes.');
-      }
+      // if (remainingBytes < 0) {
```
Comment (isoos) [inline lib/src/utils.dart:72]: Is this related?

File: test/screenshot_test.dart [hunk 2]
```diff
@@ -236,7 +236,7 @@ void main() {
     final descriptor =
         package('my_package', pubspecExtras: pubspecExtras, extraFiles: [
       d.dir('example', [
-        d.file('example.md', 'Example'),
+        d.file('EXAMPLE.md', 'Example'),
```
Comment (isoos) [inline test/screenshot_test.dart:239]: Is this related?

File: lib/src/utils.dart (hunk 1 above)
Comment (sigurdm) [reply lib/src/utils.dart]: Ah sorry, no!
I had junk in my checkout :(

**Review by isoos — APPROVED** [review]
Please also bump version + changelog.

**Review by isoos — APPROVED** [review]
